jenkinsbuilder/weekly-scan.py

```diff
@@ -109,18 +109,7 @@
         }
 
         job = bs.put(json.dumps(data))
-        # Initializing Database entries
-        project = Project.objects.get(
-            name=project_name
-        )
-        if not project:
-            print(
-                str.format(
-                    "Skipping {}, as project object was not found",
```

Comment:
@mohammedzee1000 : also we were skipping the project before putting the job for same on tube!!  Its fixed now.